lib/board.rb

```diff
@@ -5,24 +5,26 @@ class Board
               :grid
 
   def initialize
-    @header = ["A", "B", "C", "D", "E", "F", "G"]
     @rows = 6
     @columns = 7
     @grid = make_grid
```

Comment:
Future method needs to overwrite this variable so it isn't running make_grid every time.